build.sh

```diff
@@ -3,16 +3,22 @@
 function build() {
   apt update &> /dev/null
   apt install -q -y make &> /dev/null
-  make build
+  VERSION=${BUILD_VERSION} make build sonar
 }
 
 function buildLocal() {
-  make build
+  VERSION=${BUILD_VERSION} make build
 }
 
 ### MAIN ####
-VERSION=$(git describe --long --always --match='v*.*' | sed 's/v//; s/-/./')
-echo "##### BUILDING ${VERSION} ######"
+git config --global --add safe.directory /work
```

Comment:
I remember seeing this permissions error from git in the pipelines once before. Would be nice to move this line so it only happens when running in the Pipelines, but not a big deal since it works on local also